ci/builder/Dockerfile

```diff
@@ -138,13 +138,17 @@ RUN mkdir rust \
 RUN ln -s /usr/local/lib/rustlib/x86_64-unknown-linux-gnu/bin/rust-lld /opt/x-tools/x86_64-unknown-linux-gnu/bin/x86_64-unknown-linux-gnu-ld.lld \
     && ln -s /usr/local/lib/rustlib/x86_64-unknown-linux-gnu/bin/rust-lld /opt/x-tools/x86_64-unknown-linux-gnu/bin/x86_64-unknown-linux-gnu-lld
 
-# Step 2e. Set LC_CTYPE to allow UTF-8, for Python 3.6. This will become
+# Step 2e. Install the Antithesis stub instrumentation library.
+
+COPY libvoidstar.so /usr/local/lib/
```

Comment:
Ideally only the antithesis-materialized container should have this .so. Can we move this COPY step in the Dockerfile in ```test/antithesis/materialized``` ?

Reply:
Unfortunately not! It needs to be available at build time, and, AIUI, it doesn't actually need to be available at runtime, because Antithesis will swap in the actual library at runtime. (This one is just a stub library.)